jerry-debugger/jerry-client-ws.html

```diff
@@ -0,0 +1,970 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>
+JerryScript HTML (websocket) debugger-client
+</title>
+<style>
+body {
+  text-align: center;
+}
+textarea {
+  margin-bottom: 30px;
+}
+input {
+  margin-top: 10px;
+  width: 657px;
+}
+</style>
+</head>
+<body>
+
+<h2>JerryScript HTML (websocket) debugger-client</h2>
+
+<textarea id="log" rows="20" cols="80"></textarea><br>
+
+Getting help: type 'help' in the command line below.<br>
+
+<input id="command" type="text" onkeypress="return debuggerCommand(event)">
```

Comment:
If we always return true on keypress then maybe something like this would be a bit better:
``` debuggerCommand(event); return true; ```